src/applications/check-in/api/versions/v2.js

```diff
@@ -3,12 +3,20 @@ import environment from 'platform/utilities/environment';
 import { makeApiCallWithSentry } from '../utils';
 
 const v2 = {
-  getSession: async ({ token, checkInType }) => {
+  getSession: async ({
+    token,
+    checkInType,
+    isLorotaSecurityUpdatesEnabled = false,
+  }) => {
     const url = `/check_in/v2/sessions/`;
     const checkInTypeSlug = checkInType ? `?checkInType=${checkInType}` : '';
+    const eventLabel = `${checkInType ?? 'day-of'}-get-current-session-${
```

Comment:
is the checkInType stuff here redundant of whats already in the slug>

Reply:
hmm I don't think so, this is what I see in GA currently:
![Screen Shot 2022-07-21 at 10 15 46](https://user-images.githubusercontent.com/101649/180263249-39c170a2-c5d3-4239-bf12-35a9c1d8c63b.png)

Reply:
and: 
![Screen Shot 2022-07-21 at 10 16 25](https://user-images.githubusercontent.com/101649/180263412-d886bc00-3481-4c30-a21a-725df905073b.png)

Reply:
ah ok, these things translate weird into GA

Reply:
![](https://media4.giphy.com/media/pls8xeXFbASfGMn6jD/giphy.gif)